shardingsphere-infra/shardingsphere-infra-optimize/src/main/java/org/apache/shardingsphere/infra/optimize/core/convert/converter/SqlNodeConverterUtil.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.infra.optimize.core.convert.converter;
+
+import com.google.common.collect.Lists;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.shardingsphere.infra.optimize.core.convert.converter.impl.ColumnOrderByItemSqlNodeConverter;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.order.item.ColumnOrderByItemSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.order.item.ExpressionOrderByItemSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.order.item.IndexOrderByItemSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.order.item.OrderByItemSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.order.item.TextOrderByItemSegment;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Utility for sql node converter.
+ */
+public final class SqlNodeConverterUtil {
+    
+    /**
+     * Convert order by items.
+     * @param orderByItems order by item list.
+     * @return a collection of order by item <code>SqlNode</code>
+     */
+    public static List<SqlNode> convertOrderByItems(final Collection<OrderByItemSegment> orderByItems) {
+        List<SqlNode> sqlNodes = Lists.newArrayList();
```

Comment:
Hey, there only two places to call this class, right? Do you think there will be more classes calling this one later on?

Reply:
This PR now looks great, after we fix this one, it is time to merge it, I suppose.

Reply:
Please give a look at the failed ci tests.

Reply:
I suppose you still have this sense that this class is supposed to implement the converting interface, just because its result is not a sqlNode, we have this util class. If there are only two classes calling this one, I guess it is acceptable to do two `for each`  loops. How do you think?

Reply:
By now, there are only two places calling this method, `SqlNodeConverterUtil.convertOrderByItems `. 
I don't like duplicate code snippets. If a code block should be invoked in more than one method, I prefer to move this code block into a single method in the same class or in another util class.

Reply:
> Please give a look at the failed ci tests.

@tristaZero  I fixed the failed ci test.

Reply:
Ok, It's about ones' design thinking, you can keep your style.